src/less/components/homeLink.less

```diff
@@ -0,0 +1,5 @@
+.HomeLink {
```

Comment:
I think we should just remove this `.less` file for now. This should probably render in place acting almost like a bread crumb. See this example - http://beta.consumerfinance.gov/the-bureau/leadership-calendar/

![home-link](https://cloud.githubusercontent.com/assets/2932572/14207163/48bab4b4-f7e6-11e5-9c67-8f9da0f58f7b.png)

We can add styling later ... to add the icon if needed.